hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -97,6 +97,11 @@ func virtualMachineTemplateBase(image string, kvPlatform *hyperv1.KubevirtNodePo
 		Spec: kubevirtv1.VirtualMachineSpec{
 			RunStrategy: &runAlways,
 			Template: &kubevirtv1.VirtualMachineInstanceTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						"kubevirt.io/allow-pod-bridge-network-live-migration": "",
+					},
+				},
```

Comment:
i suppose the risk of enabling this now is that the CNV logic that allows migration might land in a future release before the ovn-k logic, and in that case a live migration would cause the guest node to fail.

do you think it's reasonable to wait to merge this until after the ovn-k logic is merged downstream? or is there value in having this in now that's worth the risk?

Reply:
This should not be merge, is here for testing purposes.

/Hold